std/format.d

```diff
@@ -342,24 +342,24 @@ $(I FormatChar):
     Floating point infinities are formatted as $(B inf) or
     $(B infinity) if the
     $(I FormatChar) is lower case, or $(B INF) or $(B INFINITY) if upper.
+*/
```

Comment:
Closing the DDoc here and reopening it afterwards will effectively remove all of the above from the generated documentation (see doc autotester). Curiously this only affects DDoc, not DDox. Probably a weird compiler bug?